nipype/algorithms/misc.py

```diff
@@ -850,22 +853,26 @@ def _run_interface(self, runtime):
         try:
             import pandas as pd
         except ImportError:
-            raise ImportError('This interface requires pandas (http://pandas.pydata.org/) to run.')
+            raise ImportError(('This interface requires pandas '
+                               '(http://pandas.pydata.org/) to run.'))
 
         try:
             import lockfile as pl
```

Comment:
@satra, I've been looking at ```portalocker``` (http://portalocker.readthedocs.org/en/latest/usage.html#examples) included in nipype.externals, and before integrating it in this interface I have a minor concern:
  - I think that the script included in nipype.externals is a bit outdated. The new version includes the class Lock that can be used with contexts. Therefore, I would update the external code to this new version.

I would proceed with this PR as it fixes a bug on this interface. Then update portalocker and finally integrate it in all io classes.